src/components/fileviewercov.js

```diff
@@ -1,6 +1,8 @@
 /* This file contains coverage information for a particular revision of a source file */
 import React from 'react';
 
+import * as Colour from '../utils/colour';
```

Comment:
Although "colour" is spelt correctly, we should not use it in code: The rest of the world is expected to use American english, and we Canadians do not get an exception. You *can* use this spelling if it is contained in a module, or a method, where it has limited scope; so `Colour` can be tolerated.  Using the name across module boundaries should not be done; `/utils/colour` should be fixed.

Consider a situation where we are looking for a colour variable in our code base; Using `colour` makes it harder, or impossible for a third party to find what they are looking for.

Reply:
Thanks Kyle for the insight, I've never considered that this would make it harder for others to search what they are looking for.